<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgMusicProperties</class>
 <widget class="QDialog" name="DlgMusicProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>331</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Sans Serif</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Music properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="margin">
    <number>8</number>
   </property>
   <item>
    <widget class="QRadioButton" name="SameMusicRD">
     <property name="toolTip">
      <string>You can continue the playlist defined in a previous slide</string>
     </property>
     <property name="text">
      <string>Continue current playlist (defined in a previous slide)</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>30</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>2</number>
     </property>
     <item row="1" column="0">
      <widget class="QRadioButton" name="SameMusicReduceVolumeRD">
       <property name="toolTip">
        <string>Specifies that from now on, the sound volume is reduced.
This option is particularly useful, for example, to reduce the sound of the music during the play of a video.
Remark: This volume reduction is applied only for the selected slide.</string>
       </property>
       <property name="text">
        <string>Change volume level to :</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_2</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="VolumeReductionFactorCB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Define the sound volume for this slide</string>
       </property>
       <property name="currentIndex">
        <number>7</number>
       </property>
       <item>
        <property name="text">
         <string notr="true">90</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">80</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">70</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">60</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">50</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">40</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">30</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">20</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">10</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string notr="true">%</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QRadioButton" name="SameMusicPauseRD">
       <property name="toolTip">
        <string>Pauses the current play list.
Remark: This pause is applied only for the selected slide</string>
       </property>
       <property name="text">
        <string>Set to pause</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_2</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QRadioButton" name="SameMusicNormalRD">
       <property name="toolTip">
        <string>No specific settings</string>
       </property>
       <property name="text">
        <string>Normal</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_2</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QRadioButton" name="NewMusicRD">
     <property name="toolTip">
      <string>You can start a new playlist from this slide</string>
     </property>
     <property name="text">
      <string>Start a new playlist from this slide</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="AddMusicBt">
       <property name="toolTip">
        <string>Adds a file to the playlist.
A file selection dialog appears, allowing you to select a file</string>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset resource="../../../RSCffDiaporama.qrc">
         <normaloff>:/img/object_sound.png</normaloff>:/img/object_sound.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="RemoveMusicBt">
       <property name="toolTip">
        <string>Removes the current selected file from the playlist</string>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset resource="../../../RSCffDiaporama.qrc">
         <normaloff>:/img/trash.png</normaloff>:/img/trash.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="UpMusicBt">
       <property name="toolTip">
        <string>Changes the order in which the music files will be played.
Moves the currently selected file one step up.</string>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset resource="../../../RSCffDiaporama.qrc">
         <normaloff>:/img/arrow_up.png</normaloff>:/img/arrow_up.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="DownMusicBt">
       <property name="toolTip">
        <string>Changes the order in which the music files will be played.
Moves the currently selected file one step down</string>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset resource="../../../RSCffDiaporama.qrc">
         <normaloff>:/img/arrow_down.png</normaloff>:/img/arrow_down.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Label1">
       <property name="text">
        <string>Total playlist duration : </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="LabelDuration">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string notr="true">00:00:00</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="PlayListTable">
     <property name="toolTip">
      <string>The music files are listed in the order in which they will be played, from top to bottom.
The Volume column defines the &quot; Master volume &quot; for the file.
This volume is independent of any reductions in the volume that you define in the upper part of the dialog</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>File</string>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Duration</string>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Volume</string>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Source folder</string>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QToolButton" name="HelpToolTips">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The Music properties dialog allows you to specify the music track.&lt;/p&gt;&lt;p&gt;The main functional principle for music tracks is that as long as you do not change the settings, the previous settings apply. So, if:&lt;br/&gt;- Slide 1 – Defines a play list&lt;br/&gt;- Slide 2 – Contains no settings&lt;br/&gt;- Slide 3 – Contains no settings&lt;br/&gt;- Slide 4 – Defines a play list&lt;br/&gt;- Slide 5 – Contains no settings&lt;/p&gt;&lt;p&gt;Then:&lt;br/&gt;- Slide 2 and 3 continue to play the play list defined in slide 1&lt;br/&gt;- Slide 5 continues to play the play list define in slide 4&lt;/p&gt;&lt;p&gt;Available settings for the music track :&lt;br/&gt;- Continue current playlist: Continues to play the play list defined by a previous slide&lt;br/&gt;- Start a new playlist: Allows you to create a new playlist. If a list defined in a previous slide, is still playing, it is stopped with a fade-out effect (progressive decline of the sound volume up to complete stop).&lt;/p&gt;&lt;p&gt;Remark:&lt;/p&gt;&lt;p&gt;The fade in/out effects and the increase/decrease of volume take place during the entering slide transitions. Consequently:&lt;br/&gt;- They last for the same time as the entering transition.&lt;br/&gt;- If a slide has an entering transition set to &amp;quot;No transition&amp;quot;, the sound effect is not progressive but instantaneous.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton {
	background-color: transparent;
	border: none; 
    font-size: 9px;
}
</string>
       </property>
       <property name="text">
        <string notr="true">?</string>
       </property>
       <property name="icon">
        <iconset resource="../../../RSCffDiaporama.qrc">
         <normaloff>:/img/help.png</normaloff>:/img/help.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="OKBT">
       <property name="toolTip">
        <string>Accept modification and exit dialog box</string>
       </property>
       <property name="text">
        <string>&amp;Ok</string>
       </property>
       <property name="icon">
        <iconset resource="../../../RSCffDiaporama.qrc">
         <normaloff>:/img/action_ok.png</normaloff>:/img/action_ok.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CancelBt">
       <property name="toolTip">
        <string>Cancel modification and exit dialog box</string>
       </property>
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../../../RSCffDiaporama.qrc">
         <normaloff>:/img/action_cancel.png</normaloff>:/img/action_cancel.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>SameMusicRD</tabstop>
  <tabstop>SameMusicNormalRD</tabstop>
  <tabstop>SameMusicReduceVolumeRD</tabstop>
  <tabstop>VolumeReductionFactorCB</tabstop>
  <tabstop>SameMusicPauseRD</tabstop>
  <tabstop>NewMusicRD</tabstop>
  <tabstop>AddMusicBt</tabstop>
  <tabstop>RemoveMusicBt</tabstop>
  <tabstop>UpMusicBt</tabstop>
  <tabstop>DownMusicBt</tabstop>
  <tabstop>PlayListTable</tabstop>
  <tabstop>OKBT</tabstop>
  <tabstop>CancelBt</tabstop>
 </tabstops>
 <resources>
  <include location="../../../RSCffDiaporama.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
